content/scalers/mysql.md

```diff
@@ -0,0 +1,101 @@
++++
+fragment = "content"
+weight = 100
+title = "MySQL"
+background = "light"
++++
+
+Scale applications based on MySQL query result.
+
+**Availability:** v1.1+ | **Maintainer:** Community
+
+<!--more-->
+
+### Trigger Specification
+
+This specification describes the `mysql` trigger that scales based on result of MySQL query.
+The trigger requires the following information:
+- `query` - a MySQL query that should return single numeric value that will be used for HPA
+- `queryValue` - a threshold that is used as `targetAverageValue` in HPA. 
+- Connection information as `dsnConnectionStr` or set of required params (more information below)
+
+#### Using `dsnConnectionStr`
+One option to connect to MySQL database is to use dsn connection string from environment variable:
+```yaml
+triggers:
+  - type: mysql
+    metadata:
+      dsnConnectionStr: "SQL_CONN_STR" // SQL_CONN_STR env variable should point to valid connection string
+      queryValue: "4"
+      query: "SELECT CEIL(COUNT(*) / 6) FROM task_instance WHERE state='running' OR state='queued'"
+```
+
+#### Using authentication parameters
+Other possibility is to pass all required parameters to build a new connection string:
+```yaml
+triggers:
+  - type: mysql
+    metadata:
+      username: "root"
+      password: "MYSQL_PASSWORD" // MYSQL_PASSWORD env variable should point to the password
+      host: "mysql"
+      port: "3306"
+      dbname: "stats_db"
+      queryValue: "4"
+      query: "SELECT CEIL(COUNT(*) / 6) FROM task_instance WHERE state='running' OR state='queued'"
+```
+In this case the following additional parameters must be passed:
+- `username` used to access MySQL database
+- `password` used for the given user, this should be blank (no password) or point to an environment
+ variable with the password
+- `host` and `port` of the database
+- `dbname` as name of the database
+
+### Example
+
+Here is an example of how to deploy a scaled object with the `mysql` scale trigger which uses
```

Comment:
Personally I'd go as secure/simple as possible and use `connectionString` approach here, thoughts?

Reply:
I am ok with that ;)